### DISCUSSION TITLE
Native link by default?

### DISCUSSION THREAD

**Discussion by cocochepeau** [pinecone-router/router#39] [Q&A] [answered]
Hey there,
thanks for the wonderful alpinejs router! It works great and I plan to use it on production soon. :)

I have just one request/how to question:
**how can we invert the "native" link feature? Meaning, by default links would not be handled by the router, and fallback to native html links.**

I think it could be a nice feature since alpinejs is not always used for full app sites but rather internal page components (like i'm doing).

Best,

**Comment by rehhouari** [comment]
Good idea! I'll try to implement it in settings

**Comment by rehhouari** [comment, marked as answer]
Hello, I just published version `4.4.0` with the feature you suggested!
It should be working well, since it wasn't that complicated, but it would be great if you could test it out!

[Here is the documentation link](https://github.com/pinecone-router/router?tab=readme-ov-file#disable-link-handling-globally), but TLDR:
- Set `window.PineconeRouter.settings.interceptLinks = false`
- Put an empty `x-link` attribute to the link which you want to handle
- - Or alternatively through Javascript using `$router.navigate()` inside an alpine component or `PineconeRouter.context.navigate('/path')` anywhere.

**Reply by cocochepeau** [reply]
Great! thanks for the quick change. I'll try this out tomorrow morning 👌  and will get back to you.

Best,

**Reply by cocochepeau** [reply]
Getting back to you sooner than expected! It works. :)

Thanks again.

**Reply by rehhouari** [reply]
great to hear!
